common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -112,6 +112,11 @@ object Messages {
                                     pkgSize: Long, pkgChecksum: String,
                                     timestamp: Instant = Instant.now()) extends BusMessage
 
+  final case class DeltaGenerationRequest(id: Uuid, fromVersion: String, toVersion: String) extends BusMessage
+
+  final case class DeltaGenerated(campaignId: Uuid, fromVersion: String, toVersion: String, size: Long,
+                                  succeeded: Boolean) extends BusMessage
```

Comment:
Well the type `Uuid` is also not in libats-messaging, but the important property is that the field serialises to an `UUID`. So you could use a `Campaign.Id` here if you want to.